build.gradle

```diff
@@ -94,6 +92,14 @@ allprojects {
                 name = "build"
                 url = "$buildDir/repo"
             }
+
+            // Artifacts properties - Set the properties that should be used by all artifacts
+            publications {
+                maven(MavenPublication) {
+                    groupId = "com.r3.conclave"
+                    version = conclave_graal_version
```

Comment:
Can you not set these inside `allprojects` block above? We do this in common.gradle in the SDK repo. By doing this don't need this `publications` block.

Reply:
I am not sure if I am following. This is the graal-patches project. Don't we need to set the groupId and version for the Maven coordinates? Besides this is a comment from above:
![image](https://user-images.githubusercontent.com/6329837/181197177-a13b18e7-6d72-4ae4-8004-f5aa9a79814d.png)

Reply:
Let me raise a PR instead.